source/administration/changelog.md

```diff
@@ -4,6 +4,118 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Added support for webhook message attachments to trigger mentions.
+ - Stripped markdown formatting characters from desktop notifications and "Commented on..." text.
+ - Added ability to bulk import emoji.
+ - Added support for file attachments in bulk import.
+ 
+#### New Plugins (All Beta)
+ - [Antivirus plugin](https://github.com/mattermost/mattermost-plugin-antivirus) to scan for viruses before uploading a file to Mattermost. Supports [ClamAV anti-virus software](https://www.clamav.net/) across browser, Desktop Apps and the Mobile Apps.
+ - [GitHub plugin](https://github.com/mattermost/mattermost-plugin-github) to subscribe to notifications, and to keep track of unread GitHub messages and open pull requests requiring your attention.
+
+#### Server Plugins: Release Candidate
+ - A release candidate (RC) is released for server plugins. Stable release is expected in v5.3 or v5.4.
+ - Added various API methods for plugins to provide the same capabilities as the REST API.
+ - Added support to intercept file uploads before the file is uploaded to a Mattermost server.
+ - Added support for plugins to respond after a user joins/leaves a channel or a team, or creates a new channel.
+ - Added support for plugins to respond prior to or after a user logs in to a Mattermost server.
+ - Added support for plugins to update user status. Sample use case is setting a user’s status to Do Not Disturb based on Google Calendar events.
+ - Added CSRF tokens that are attached to users sessions. The tokens can be enforced as an alternative to XHR checks in the plugin request system.
+ - Added session token to context for ServeHTTP hook.
+
+#### Webapp Plugins: Beta
+ - Upcoming Mattermost UI redesign may cause breaking changes to webapp plugins. Hence, webapp plugins remain as beta in v5.2.
+ - Added support to override [...] post menu, and paperclip icon for file uploads.
+ - Added support for multiple plugins to add components at the same integration points instead of only allowing one plugin to do so.
+ - Removed ability to fully override profile popover. Instead, multiple plugins can now add to the profile popover via multiple integration points.
+ - For an up-to-date list of pluggable UI components, [see this list in our demo plugin](https://github.com/mattermost/mattermost-plugin-demo/tree/master/webapp#components).
+
+#### Notifications
+ - Added localized timestamp support for email notifications.
+ 
+#### Administration
+ - In the compliance export status table, in System Console > Compliance > Compliance Export, added a number of exported records to Details column.
+ - Added support for cross-origin resource sharing.
+
+#### Command Line Interface (CLI)
+ - Enhanced log output from Permanent Delete CLI command to delete FileInfos for a user's posts.
+ - Addded channel renaming to CLI.
+ 
+#### Enterprise Edition
+ - Added the Global Relay Export CLI command.
+ - Added support to search message attachment/plugin contents.
```

Comment:
probably remove "message attachment" as that might get confused with file uploads